test/saithrift/tests/saiacl.py

```diff
@@ -15,114 +15,382 @@
 """
 Thrift SAI interface ACL tests
 """
-
 from switch import *
 import sai_base_test
 
+# Define test data
+SWITCH_MAC =   "00:77:66:55:44:33"
+TEST_SRC_MAC = "00:22:22:22:22:22"
+NEIGHBOR_MAC = "00:11:22:33:44:55"
+IP_SRC =       "192.168.0.1"
+IP_DST =       "10.10.10.1"
+IP_MASK =      '255.255.255.0'
+IP_ID =        105
+IP_TTL =       64
+PORT_IN =      2
+PORT_OUT =     1
+PORT_CPU =     176
```

Comment:
@lguohan  -- i agree this should be CPU PORT ID -- which cannot be hard coded. @garyachy : Please update your PR.